crates/iota-sdk-types/src/crypto/validator.rs

```diff
@@ -161,14 +161,13 @@ pub struct ValidatorSignature {
     pub signature: Bls12381Signature,
 }
 
-#[cfg(test)]
+#[cfg(all(feature = "serde", test))]
```

Comment:
Was this actually failing? Seems a bit annoying if we ever add more tests, we may forget to remove the serde feature